src/tools/quemap/lightmap.c

```diff
@@ -75,8 +75,13 @@ static void BuildLightmapExtents(lightmap_t *lm) {
 		AddStToBounds(st, lm->st_mins, lm->st_maxs);
 	}
 
+	#if 1 // bilinear or bicubic filtering
+	lm->w = floorf(lm->st_maxs[0] - lm->st_mins[0]) + 4;
+	lm->h = floorf(lm->st_maxs[1] - lm->st_mins[1]) + 4;
+	#else // only bilinear filtering
 	lm->w = floorf(lm->st_maxs[0] - lm->st_mins[0]) + 2;
 	lm->h = floorf(lm->st_maxs[1] - lm->st_mins[1]) + 2;
+	#endif
```

Comment:
Feel free to simply change this to the oversampled size that you need, no need for #if.